src/sentry/search/events/constants.py

```diff
@@ -352,7 +356,13 @@ class ThresholdDict(TypedDict):
 SPAN_METRIC_DURATION_COLUMNS = {
     key
     for key, value in SPAN_METRICS_MAP.items()
-    if value.endswith("@millisecond") and value.startswith("d:")
+    if (value.endswith("@millisecond") and value.startswith("d:"))
```

Comment:
do we need to do the `value.startswith(...)` check? can't we just look at the unit portion of the mri?

Reply:
I don't think so. I kept them because I wanted to make my changes as isolated as possible, but I think it's totally fine to look at the unit portion here. Updated.